Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+         //not sure how to call functions within here.  Test says it cannot find them.
```

Comment:
"Test says it cannot find them"- Demo method is not tested, it's just for you, to run from the console and see if it works. I presume that's what you meant, if not- let me know.

To run a function from a function is quite simple- same rules apply (and you actually already do it)
```
// person 1
var age = PromptInt("Please enter age");
...
// person 2
...
```
person1 and person2 processing should ideally be a function as well. Same function called twice.

Let me know if it's clear how to proceed. :)

Reply:
Ok.  Took a couple days and came back to this with a fresh mind.  I think this is what you're looking for.

I think I was spending too much time on just trying to figure this out, that my mind just wasn't clicking after a few hours.